guides/doc-Upgrading_and_Updating/topics/upgrading_a_disconnected_satellite.adoc

```diff
@@ -119,6 +119,14 @@ Do not install or update any packages at this stage.
 ----
 baseurl=file:///media/sat6/
 ----
+... Optional: If you want to configure a custom path for the Ansible Runner packages, navigate to the `Ansible` repository and modify the `baseurl` directive with your custom path.
```

Comment:
the BZ is about ansible, which needs to be set up identically to the SCL below (with the caveat, that there seem not to be an ansible.repo file on the ISO). and it's not optional.

Reply:
Thank you!